.azure-pipelines-templates/deploy_attestation_container.yml

```diff
@@ -58,15 +58,34 @@ jobs:
     displayName: "Cleanup Attestation Container"
     pool:
       vmImage: ubuntu-20.04
-    dependsOn: ${{ parameters.used_by }}
+    dependsOn:
+      - deploy_attestation_container
+      - ${{ parameters.used_by }}
     condition: always()
+    variables:
+      IpAddresses: $[ dependencies.deploy_attestation_container.outputs['deploy_attestation_container.ipAddresses'] ]
+      sshKey: $[ dependencies.deploy_attestation_container.outputs['generate_ssh_key.sshKey'] ]
     steps:
       - template: azure_cli.yml
         parameters:
           app_id: $(ATTESTATION_CONTAINER_CI_APP_ID)
           service_principal_password: $(ATTESTATION_CONTAINER_CI_SERVICE_PRINCIPAL_PASSWORD)
           tenant: $(ATTESTATION_CONTAINER_CI_TENANT)
 
+      - template: install_ssh_key.yml
+        parameters:
+          ssh_key: $(sshKey)
+
+      - script: |
+          set -ex
+          BUILD_NUMBER=$(echo $(Build.BuildNumber) | sed 's/\.//g')
+          IP_ADDR=$(echo "$(IpAddresses)" | awk -v container_group="ci-$BUILD_NUMBER-business-logic-0" '$1 == container_group {print $2}')
+          ssh agent@$IP_ADDR -o "StrictHostKeyChecking no" '
+          rm -rf /acci/workspace_$(Build.BuildNumber)
```

Comment:
Is this needed? The workspace directory is usually where the CCF nodes are started but that shouldn't be the case with the deployment of the attestation container?